### ISSUE TITLE
Specify bibencoding per bib file

### ISSUE THREAD

**Issue by moewew** [plk/biber#217]
Inspired by https://tex.stackexchange.com/q/420103/35864

Would it be possible to specify the `bibencoding` per-`.bib`-file via the optional argument to `addbibresource`? If a user uses `.bib` files from different sources this may come in very handy.

The following MWE will not work as expected
```latex
\documentclass[british]{article}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage{babel}
\usepackage{csquotes}

\usepackage[style=authoryear, backend=biber]{biblatex}

\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}
@article{steendam,
  author   = {H. Steendam and T. Q. Wang and J. Armstrong},
  title    = {Theoretical Lower Bound for Indoor Visible Light Positioning Using Received Signal Strength Measurements and an Aperture-Based Receiver},
  journal  = IEEE_J_JLT,
  date     = {2017-01},
  volume   = {35},
  number   = {2},
  pages    = {309-319},
  doi      = {10.1109/JLT.2016.2645603},
}
\end{filecontents}

\addbibresource{IEEEabrv.bib}
\addbibresource{\jobname.bib}

\begin{document}
\cite{steendam}
\printbibliography
\end{document}
```
since `IEEEabrv.bib` is not actually UTF-8 encoded. It uses Latin-1 instead because of a stray `á` in the comments. A `\addbibresource[bibencoding=latin1]{IEEEabrv.bib}` would help here, because we still want all other files to be in UTF-8.

**Comment by plk** [comment]
Implemented in 2.12/3.12 in DEV

**Comment by moewew** [comment]
This works brilliantly. Thank you very much.
